extensions/default/src/commandsModule.ts

```diff
@@ -432,14 +432,14 @@ const commandsModule = ({
         // This findOrCreateViewport returns either one of the updatedViewports
         // returned from the HP service OR if there is not one from the HP service then
         // simply returns what was in the previous state.
-        const findOrCreateViewport = (viewportIndex: number) => {
+        const findOrCreateViewport = (pos: number) => {
           const viewport = updatedViewports.find(
-            viewport => viewport.viewportIndex === viewportIndex
+            viewport => viewport.pos === pos
```

Comment:
This is a bit weird because from what I can tell, a viewport is not storing its `pos` anywhere... correct? So I think this check no longer makes sense.  Furthermore, I think this check is from a time when somebody could toggle to one up, swap in a different series and then return to the previous layout and the swapped in series would occupy the viewport that was toggled to/from one up. From what I can tell that no longer works, so if we don't care about that feature then I think even the call to `getViewportsRequireUpdate` above is no longer necessary.

Reply:
I don't think right now we have the feature you are saying

> somebody could toggle to one up, swap in a different series and then return to the previous layout and the swapped in series would occupy the viewport that was toggled to/from one up

Can you try it in the https://viewer-dev.ohif.org/viewer?StudyInstanceUIDs=1.3.6.1.4.1.14519.5.2.1.7009.2403.334240657131972136850343327463

Reply:
Well the feature used to be there, but this PR was not the one that removed it.

That aside, this check for `viewport.pos === pos` does not really make sense because from what I can see, we are NEVER saving/storing this position per viewport and furthermore `HangingProtocolService.getViewportsRequireUpdate` does NOT return it (i.e. `pos`) as one of the fields so I am confused as to why we even need to check for `updatedViewports`. It is likely all we have to do in `findOrCreateViewport` is to return the previous state.